packages/components/src/InputFile/InputFile.tsx

```diff
@@ -103,10 +103,18 @@ interface InputFileProps {
    *
    * @param "images" - only accepts all types of image
    * @param "basicImages" - only accepts png, jpg and jpeg
+   * @param "custom" - uses `acceptedMIMETypes` as the list of allowed types
    *
    * @default "all"
    */
-  readonly allowedTypes?: "all" | "images" | "basicImages";
+  readonly allowedTypes?: "all" | "images" | "basicImages" | "custom";
```

Comment:
What if instead of custom, you allow the string[] that way, you won't need the other type

Reply:
That also works. I will update the implementation

Reply:
Looks like this was addressed in https://github.com/GetJobber/atlantis/pull/1708/commits/1957cef5bce8df14081fb3350b3fbf178755c6fe!